api/v1alpha1/channel_types.go

```diff
@@ -42,6 +42,14 @@ type ChannelSpec struct {
 	// Topic of the channel
 	// +optional
 	Topic string `json:"topic,omitempty"`
+
+	// Don't remove users if they aren't listed.
+	// +optional
+	SkipRemoveUsers bool `json:"skipRemoveUsers,omitempty"`
+
+	// Don't delete the channel when the object is deleted
+	// +optional
+	SkipDeleteChannel bool `json:"skipDeleteChannel,omitempty"`
```

Comment:
```suggestion
	SkipArchiveChannel bool `json:"skipArchiveChannel,omitempty"`
```
can it be changed to `archive` to be consistent with the terminologies? Rest everything looks fine.

Reply:
Shoot. I have this deployed all over the place already. If I get some time I will change it, but unlikely :(